style.css

```diff
@@ -0,0 +1,657 @@
+* {
+  margin: 0;
+  padding: 0;
+  font-family: Arial, Helvetica, sans-serif;
+  box-sizing: border-box;
+}
+
+header {
+  display: flex;
+  position: fixed;
+  background: #fff;
+  z-index: 2;
+  top: 0;
+  right: 0;
+  width: 100%;
+  flex-direction: column;
+}
+
+main {
+  position: relative;
+  top: 105px;
+}
+
+.logo {
+  display: flex;
+  height: 100%;
+}
+
+section ul {
+  margin-left: 10px;
+  list-style: none;
+}
+
+.logo ul {
+  display: flex;
+  width: 100%;
+  justify-content: flex-end;
+}
+
+section ul li {
+  border-bottom: 1px solid rgb(185, 184, 184);
+}
+
+main footer ul li {
+  display: flex;
+}
+
+.logo ul li {
+  display: flex;
+  padding: 5px 5px;
+  width: 50px;
+  border-left: 1px solid #ccc;
+  justify-content: center;
+}
+
+h1,
+h2,
+h2 i,
+h1 i {
+  color: #f42;
+}
+
+main footer ul li a i {
+  color: white;
+  font-size: 2rem;
+}
+
+.logo ul li i {
+  font-size: 20px;
+  cursor: pointer;
+}
+
+.logo ul li #menu {
+  color: #f42;
+}
+
+.section-content img {
+  height: 300px;
+  width: 100%;
+  opacity: 0.5;

```

Comment:
You can try playing with [`object-fit`](https://www.w3schools.com/css/css3_object-fit.asp) to avoid the image being stretch, also maybe try letting the height be responsive by removing it or trying another height so it resembles more the original page.

Reply:
Hello!
The logo is not visible on the large version. Is visible if we scroll down I don't know how to make it with HTML and CSS only

Reply:
Hi!

The logo that is missing is not the one from the __Navigation bar__, is the one from the __Top 3 articles section__:

![image](https://user-images.githubusercontent.com/2121481/73870990-fed5de80-481a-11ea-9252-b3d5fa23fa6c.png)

Do not worry about the one that only appears once you scroll down o large screens 👍.